### PR TITLE
added jv-oop homework solution

### PR DESCRIPTION
**Pull request by vitaliiplys**

### CODE REVIEW

**Review by kozhukhovsky — CHANGES_REQUESTED** [review]
Java Cl checks failed. Before making a Pull Request (or a new commit to an already existing PR), you need to write the `mvn clean package` command in the terminal and make sure that all checks pass. After that, the mentor will be able to review the homework

Solution is ok, but need some codestyle fixes

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
```
Comment (olekskov) [inline src/main/java/core/basesyntax/Bulldozer.java:3]:
```suggestion
public class Bulldozer extends Machine {
```

File: src/main/java/core/basesyntax/Excavator.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
```
Comment (olekskov) [inline src/main/java/core/basesyntax/Excavator.java:3]:
```suggestion
public class Excavator extends Machine {
```

File: src/main/java/core/basesyntax/Truck.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
```
Comment (olekskov) [inline src/main/java/core/basesyntax/Truck.java:3]:
```suggestion
public class Truck extends Machine {
```

**Review by olekskov — APPROVED** [review]
Good job, but please pay attention to code style consitency in future (highlighted in comments)

**Review by olekskov — CHANGES_REQUESTED** [review]
Please run "mvn clean package" command locally and fix errors

**Pull request closed without merging**